client/components/restName.js

```diff
@@ -0,0 +1,103 @@
+const restName = [
```

Comment:
I didn't know you could use export default on the server side. I thought you had to use module.exports. Does this work fine?

Reply:
@IsaacLeeWebDev yeah , it works, i see the data name and id of restaurant